lib/active_merchant/billing/gateways/elavon.rb

```diff
@@ -113,10 +115,12 @@ def credit(money, creditcard, options = {})
       end
 
       def verify(credit_card, options = {})
-        MultiResponse.run(:use_first_response) do |r|
-          r.process { authorize(100, credit_card, options) }
-          r.process(:ignore_result) { void(r.authorization, options) }
-        end
+        form = {}
+        add_creditcard(form, credit_card)
+        add_address(form, options)
+        add_test_mode(form, options)
+        add_ip(form, options)
+        commit(:verify, 0, form, options)
```

Comment:
Because of the structure of the `commit` method, passing the `verify` method requires sending a $0 value for the `money` field. The remote tests are showing that sending along the unused `amount` field is fine for the `verify` method.